src/api/lib/data.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs'
+import { DB, ServiceKey } from '../types/admin'
+
+const data: DB = {
+  serviceKeys: []
+}
+
+const getData = (serviceKeys: string) => {
+  if (serviceKeys) {
+    fs.readFile('./data.db', (err, file) => {
+      if (err) {
+        return console.log('error', err)
```

Comment:
if you want to `console.log`, you should be more clear: `console.log('error reading db file', err)`